src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(operation, operand) {
```

Comment:
Better name for function that we put inside `operate` method is `callback`


> A callback is a function passed as an argument to another function.

Learn more about callback  --> [link](https://www.w3schools.com/js/js_callback.asp) 

```suggestion
    operate(callback, operand) {
```